Makefile

```diff
@@ -0,0 +1,106 @@
+# Variables
+COVERAGE_ENV = CARGO_INCREMENTAL=0 RUSTFLAGS='-Cinstrument-coverage' LLVM_PROFILE_FILE=$(shell pwd)/target/profraw/cargo-test-%p-%m.profraw
+# FIXME: In latest 0.8.19+ -t CLI argument can accept comma separated list of custom output types, hence, no need in double invocation
+GRCOV_CALL = grcov ./target/profraw --binary-path ./target/release/deps/ -s . --branch --ignore-not-existing --ignore "**/tests/**"
+
+# Install test dependencies
+install-test-deps:
+		@echo ""
+		@echo "Installing the test dependencies."
+		@echo ""
+		rustup component add llvm-tools-preview
+		# https://nexte.st/book/pre-built-binaries.html#using-nextest-in-github-actions
+		# FIXME: update to 0.9.68 when we get rid of 1.71 and 1.72.
+		cargo install cargo-nextest@=0.9.67 --locked
+		# FIXME: latest 0.8.19+ requires rustc 1.74+
+		cargo install grcov@=0.8.13 --locked
+		@echo ""
+		@echo "Test dependencies installed."
+		@echo ""
+
+# Default target
```

Comment:
Should we make `release` the default target?
I don't have a strong opinion, but running `make` will always clean the cargo workspace, which might not be the default behavior a user expects? Up to you.

Reply:
`release` target will be the default one (without the cleaning).